### DISCUSSION TITLE
chronark template

### DISCUSSION THREAD

**Discussion by cth4n** [vercel/community#3979] [Help]
### Summary

I am using the chronark template to make myself a portfolio website but I error in app/projects/page.tsx

  13 | export default async function ProjectsPage() {
  14 | 	const views = (
> 15 | 		await redis.mget<number[]>(
     | 	^
  16 | 			...allProjects.map((p) => ["pageviews", "projects", p.slug].join(":")),
  17 | 		)
  18 | 	).reduce((acc, v, i) => {
  
  Error: ERR wrong number of arguments for 'mget' command

### Steps to Reproduce

Just happens whenever I run the code.

**Comment by amyegan** [comment]
Hey @zinfromsynergy. I'm not familiar with that template. Can you share a link to it?

**Comment by cth4n** [comment]
I ended up figuring out why, i was on local like an idiot and forgot but its on the vercel website as like nextjs-portfolio-pageview-counter or something similar.

**Reply by amyegan** [reply]
😄 Glad you figured it out. I just confirmed the [Next.js Portfolio with Pageview Counter](https://vercel.com/templates/next.js/nextjs-portfolio-pageview-counter) template deploys without any errors. Please let me know if you run into any more trouble with it!
